### ISSUE TITLE
Feature Request: Joining log files into a single view.

### ISSUE THREAD

**Issue by jochenw** [zarunbal/LogExpert#54]
When watching several log files with "Follow tail", it would be nice to have all the files joined into a single view. Colors, or Markers could be used to indicate a log lines source.

**Comment by Otiel** [comment]
Isn't this the purpose of MultiFile?

**Comment by jochenw** [comment]
> Isn't this the purpose of MultiFile?

What is MultiFile, please?

**Comment by Hirogen** [comment]
Menu => File => MultiFile

**Comment by zarunbal** [comment]
Yes, @jochenw is right Mutlifile is only for rotated files where only the first file changes and the other are only moved.

This would be a really big feature, to get everything more or less right in the view displayed. My first ideas / questions about this topic (Just a quick thougth about this)

- how is decided which line should be displayed if two files gets updated at the same time
- How to handle a removed file (e.g. connection to share lost)
- What should happen if there is a full reload of a file
  - Add the lines somewhere
- How should be displayed if you open file which have already some entries in it
- Different encodings in files
- How to handle if one files is on a slow medium and the others are on the quick one.
